cli_reference/manage_cli_profiles.adoc

```diff
@@ -241,69 +231,60 @@ $ oc config view --config=<specific_filename>
 
 *Example Usage* [[example-usage]]
 
-Consider the following configuration workflow. First, set credentials for a user
-nickname *alice* that uses an
+Consider the following configuration workflow. First, login as a user
+that uses an
 xref:../architecture/additional_concepts/authentication.adoc#api-authentication[access
-token]:
+token]. This token is used by the *alice* user:
 
 [options="nowrap"]
 ----
-$ oc config set-credentials alice --token=NDM2N2MwODgtNjI1Yy10N3VhLTg1YmItYzI4NDEzZDUyYzVi
-----
-
-Set a cluster entry named *openshift1*:
-
-----
-$ oc config set-cluster openshift1 --server=https://openshift1.example.com
-----
-
-Set a context named *alice* that uses the *alice* user and the
-*openshift1* cluster:
-
-----
-$ oc config set-context alice --cluster=openshift1 --user=alice
-----
-
-Now that the *alice* context has been created, switch to that context:
-
-----
-$ oc config use-context alice
+$ oc login https://openshift1.example.com --token=ns7yVhuRNpDM9cgzfhhxQ7bM5s7N2ZVrkZepSRf4LC0
 ----
 
-Set the *aliceproject* namespace for the *alice* context:
+View the cluster entry automatically created:
 
-----
-$ oc config set contexts.alice.namespace aliceproject
-----
-
-You can now view the configuration that has been created:
-
-====
+[options="nowrap"]
 ----
 $ oc config view
 apiVersion: v1
 clusters:
 - cluster:
+    insecure-skip-tls-verify: true
     server: https://openshift1.example.com
-  name: openshift1
+  name: openshift1-example-com
 contexts:
 - context:
-    cluster: openshift1
-    namespace: aliceproject
-    user: alice
-  name: alice
-current-context: alice <1>
+    cluster: openshift1-example-com
+    namespace: default
+    user: alice/openshift1-example-com
+  name: default/openshift1-example-com/alice
+current-context: default/openshift1-example-com/alice
 kind: Config
 preferences: {}
 users:
-- name: alice
+- name: alice/openshift1.example.com
   user:
-    token: NDM2N2MwODgtNjI1Yy10N3VhLTg1YmItYzI4NDEzZDUyYzVi
+    token: ns7yVhuRNpDM9cgzfhhxQ7bM5s7N2ZVrkZepSRf4LC0
+----
+
+Update the current context to have users login to the
+*openshift* namespace:
+
+----
+$ oc config set-context --current --namespace=openshift
+----
+
+Log out and back in to confirm that the changes have taken
+effect.
+
+----
+$ oc logout
+$ oc login https://openshift1.example.com --username=alice
+...
+Using project "openshift".
```

Comment:
If you use the `Log out and back` way, here may not get `Using project "openshift"` if user alice is only normal user instead of an admin user of project "openshift"

Reply:
This section has been replaced.